lib/gssapi/lib_gssapi_loader.rb

```diff
@@ -26,16 +26,24 @@ def self.load_mit
         gssapi_lib = 'libgssapi_krb5.so.2'
         ffi_lib gssapi_lib, FFI::Library::LIBC
       when /darwin/
-        prefix = begin
-          stdout, _stderr, status = Open3.capture3("krb5-config --prefix")
-          stdout.chomp!
+        exception_message = <<BREW
+The gssapi-factual gem on macOS requires the krb5 package from Homebrew. `brew install krb5` and ensure it's in your PATH"
```

Comment:
total nitpick, but it looks like you have a `"` at the end, but i don't a matching one. should still work with the heredoc format, but not sure if you wanted to specifically quote a part of the message

Reply:
Woah you're absolutely right, thanks. It got left over from when I converted it to a heredoc.